homeassistant/components/evohome/climate.py

```diff
@@ -344,14 +385,20 @@ async def async_tcs_svc_request(self, service: str, data: dict[str, Any]) -> Non
 
     async def _set_tcs_mode(self, mode: str, until: dt | None = None) -> None:
         """Set a Controller to any of its native EVO_* operating modes."""
+
         until = dt_util.as_utc(until) if until else None
+
         await self._evo_broker.call_client_api(
-            self._evo_tcs.set_mode(mode, until=until)
+            self._evo_tcs.set_mode(mode, until=until)  # type: ignore[arg-type,unused-ignore]
```

Comment:
Note use of `unused-ignore`. The original error is:
```Argument 1 to "set_mode" of "ControlSystem" has incompatible type "str"; expected "SystemMode"  [arg-type]
```
In this case, `SystemMode` is a **StrEnum**.

Reply:
Passed in the enum instance instead of the string.